### ISSUE TITLE
Localize content (description, footer, etc.) on status pages

### ISSUE THREAD

**Issue by F0rce** [louislam/uptime-kuma#2784]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

There should be a possibility in the UI to localize description, footer, title, etc.

This would easily implement as a additional feature (completely optional), because the status page itself is already localized. It would just add the possibility to display different content specific to the language that has been „guessed" by uptime-kuma.

If this feature is somewhat relevant and usefull, I could give implementing it as a feature a try 🤝 

*Just wanted to make sure to not waste my time on an unwanted implementation*

### ✔️ Solution

There should be some kind of language-explorer in the status page settings where so called „language-keys“ should be defineable, that will be parsed to the guessed language as soon as the page loads.

As I have not looked into an simple implementation for it, it could not say what new ui elements would be needed (maybe some kind of context-menu). 

### 📝 Additional Context

Uptime-Kuma is really great and super well implemented as well as maintained. Having user-defined content localized would elevate the functionality even further.

**Comment by CommanderStorm** [comment]
@F0rce 
I cannot reproduce this issue
![image](https://github.com/louislam/uptime-kuma/assets/26258709/b4558453-fb53-4f93-8d0d-0339a919b788)
![image](https://github.com/louislam/uptime-kuma/assets/26258709/aa070924-c14d-49e0-9336-96cbd9a5acba)

The language of the status page matches the one defined in the settings
I don't know what this issue is referring to too be honest when you are talking about localising content.

I think what you want could be a duplicate of https://github.com/louislam/uptime-kuma/issues/2196

**Comment by F0rce** [comment]
> You are talking about this part, right?

Yes I am talking about the part you provided the screenshot for. In my opinion a Statuspage is visited by multiple (thousands) people all around the world. For an Website / Service that is available world-wide and thus translated to many different languages aswell, it would be nice to give context in different languages.

The current "Workaround" is setting up multiple Statuspages with different default languages (living under different subdomains). The problem with this one is, that you have to schedule an downtime in all of those instances or one of the regions would have different information than other.

Still I want to thank you for your reply @CommanderStorm - great work with Kuma!

**Comment by F0rce** [comment]
thats absolutely no worries. If it happens, it happens. If it does not reach main or is "worth" enough, that's no problem either.

Just wanted to raise this issue / feature request because I think that I am not the only one thinking of that.

Let's see what happens, cheers!
